packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -112,6 +130,25 @@ public function renderStyleProp(Environment $environment, array $props): string
         ]);
     }
 
+    /**
+     * Converts a mixed value to a string.
+     *
+     * @return string The value as a string.
+     */
+    private function getSpacingClassName(string $propName, string $propValue, ?string $breakpoint = null): string
+    {
+        $container = new ContainerBuilder();
+        $classPrefix = $container->hasParameter(SpiritWebTwigExtension::PARAMETER_SPIRIT_CSS_CLASS_PREFIX)
+            ? $container->getParameter(SpiritWebTwigExtension::PARAMETER_SPIRIT_CSS_CLASS_PREFIX)
+            : '';
+        $classPrefix = is_string($classPrefix) ? $classPrefix : '';
```

Comment:
How to avoid nested ternary 😈 

I still believe there is a better syntax than re-assinging the value. I came up with this (and Chat GPT also thinks it does the same thing):

```php
$classPrefix = $container->getParameter(SpiritWebTwigExtension::PARAMETER_SPIRIT_CSS_CLASS_PREFIX) ?? '';
```

cc @literat 

> Certainly! You can simplify the condition by using the null coalescing operator (`??`). Also, you can remove the second assignment since the null coalescing operator already handles the case where the parameter is not present or not a string:
>
>```php
>$classPrefix = $container->getParameter(SpiritWebTwigExtension::PARAMETER_SPIRIT_CSS_CLASS_PREFIX) ?? '';
>```
>
> This code checks if the parameter exists, and if it does, assigns its value to `$classPrefix`; otherwise, it assigns an empty string. It's a concise and readable way to achieve the same result as your original code.

Reply:
Approved 👍